src/FWorld.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef __FWORLD_HPP__
+#define __FWORLD_HPP__
+
+// FCharacter.cpp
+static const bool FACING_LEFT = false;
+static const float RESURRECTION_PERIOD = 3.f;
+static const float MOVEMENT_SPEED = 0.23f;
+static const float JUMPING_PERIOD = 0.4f; 
+static const FVector JUMP_VECTOR(0.f, -0.4f); // needs FVector loaded beforehand
+// AI constants skipped for now
+
+// FEngine.cpp
+static const float CREATING_POWERUP_PROBABILITY = 0.005f;
+static const FVector PROJECTTILE_SIZE(10, 10); // needs FVector loaded beforehand // what about other projectiles?
+static const float PROJECTTILE_TIME = 1000.f; // what about other projectiles?
+static const int PROJECTTILE_DAMAGE = 10; // what about other projectiles?
+// The same for powerup (heal instead of damage) in lines between 115-121, skipped for now
+
+// FGame.cpp
+// Frame rate skipped
+static const float HEALTH_BAR_SIZE_HEIGHT = 10.f; // should be local?
+static const float HEALTH_BAR_POSITON_Y = -12.f; // should be local?
+
+// FObject.cpp
+static const float MINIMUM_STILL_VELOCITY = 0.0001f;
+static const float MINIMUM_VELOCITY = 0.01f; // not sure?
+static const float DRAG = 0.9f;
+static const float GRAVITY = 8.9f;
+
+// main.cpp
+static const int WIDTH = 64;
+static const int HEIGHT = 48;
+static const int TILESIZE = 10;
```

Comment:
This again is not part of the engine (world) but rather just a specific level, thus it doesn't belong here.